cpp/include/cudf/lists/lists_column_view.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+
+namespace cudf {
+
```

Comment:
Also add a doxygen group here so this shows  up in the Modules page.
Here is an example from the `strings_column_view` header:
https://github.com/rapidsai/cudf/blob/eb896a0d75e11797d835728692146ed09e8b84d5/cpp/include/cudf/strings/strings_column_view.hpp#L30-L33
Suggest `lists_classes` perhaps as a name.
Then add the `lists_classes` to the list here:
https://github.com/rapidsai/cudf/blob/eb896a0d75e11797d835728692146ed09e8b84d5/cpp/include/doxygen_groups.h#L29-L35